### PR TITLE
Model management UI

### PR DESCRIPTION
**Pull request by popcornylu**
## Story
https://app.clubhouse.io/infuseai/story/16764/model-management-ui

## Image
image: 
~~ch16764-6498840~~ 
ch16764-4453760:  Add source run external link

### CODE REVIEW

**Comment by popcornylu** [comment]
Could we change the learn more color to white?

image

**Comment by aar0nTw** [comment]
> Could we change the learn more color to white?
> 
> image

<Tooltip
      placement='right'
      title={<span>This is used to access the MLflow artifact stores, <a style={{color: '#739af3'}} target='_blank' href='https://www.mlflow.org/docs/latest/tracking.html#artifact-stores'>Learn more</a></span>}>
      <Icon type='question-circle' />
    </Tooltip>

**Comment by aar0nTw** [comment]
> Could we change the learn more color to white?
> 
> image

Already updated

File: packages/client/src/components/share/breadcrumb.tsx [hunk 1]
```diff
@@ -26,12 +27,15 @@ const parseBreadcrumb = (match: any, basename: string, items: BreadcrumbItemSetu
...
-      const {key, matcher, title, link} = item;
+      const {key, matcher, title, link, onClick} = item;
       if (matcher && matcher.test(match.url)) {
+
         result.push(<Breadcrumb.Item  key={key}>
           <span style={{fontWeight: (last) ? 500 : 'initial', color: (last) ? 'rgba(0, 0, 0, 0.65)' : null}}>
             {
-              !!link ? <Link style={{color: (last) ? 'rgba(0, 0, 0, 0.65)' : null}} to={`${basename}${link}`}>{title}</Link> : title
+              !!link ? <Link style={{color: (last) ? 'rgba(0, 0, 0, 0.65)' : null}} to={`${basename}${link}`}>{title}</Link> :
+                !!onClick ? <a style={{color: (last) ? 'rgba(0, 0, 0, 0.65)' : null}} onClick={(e) => onClick()}>{title}</a> :
```
Comment (aar0nTw) [inline packages/client/src/components/share/breadcrumb.tsx:37]: Just use `onClick={onClick}` is more flexible for the event handler.

File: packages/client/src/ee/components/modelMngt/common.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import moment from 'moment';
+
+const parseDate = (time:string) => {
+  if (!time) {
+    return undefined;
+  }
+  const n = Number(time);
+  if (Number.isNaN(n)) {
```
Comment (aar0nTw) [inline packages/client/src/ee/components/modelMngt/common.ts:8]: use `_.isNumber` can prevent `Infinity` `-Infinity` `NaN`

File: packages/client/src/ee/containers/modelDetail.tsx [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+
+      return {
+        variables: {
+          group,
+          name: modelName
+        },
+        fetchPolicy: 'cache-and-network'
+      }
+    },
+    name: 'getModel'
```
Comment (aar0nTw) [inline packages/client/src/ee/containers/modelDetail.tsx:150]: Tips: add option `alias: 'withGetModel'` will easier determine the graphql context in react inspector.

**Review by aar0nTw — CHANGES_REQUESTED** [review]
LG, only 3 comments for changes.

File: packages/client/src/ee/components/modelMngt/common.ts [hunk 4]
```diff
@@ -0,0 +1,55 @@
+import moment from 'moment';
+import isNumber from 'lodash'
```
Comment (aar0nTw) [inline packages/client/src/ee/components/modelMngt/common.ts:2]: `import {isNumber} from 'lodash'`

**Review by aar0nTw — CHANGES_REQUESTED** [review]

File: packages/client/src/ee/components/modelMngt/common.ts (hunk 4 above)
Comment (popcornylu) [reply packages/client/src/ee/components/modelMngt/common.ts]: Done

**Review by aar0nTw — APPROVED** [review]

**Pull request merged**